src/app/components/AppNavigator/TabIcon/styles.ts

```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from 'react-native';
+import { CERULEAN, FRENCH_GREY } from '@constants/colors';
+
+const iconSize = 32;
+
+export default StyleSheet.create({
+  imageFocused: {
+    tintColor: CERULEAN,
+    height: iconSize,
+    width: iconSize
+  },
+  imageUnFocused: {
+    tintColor: FRENCH_GREY,
+    height: iconSize,
+    width: iconSize
+  }
```

Comment:
It's not necessary to repeat the same styles if only the color changes, you could use a base style and, with a conditional, change the color.
```
  baseImage: {
    tintColor: FRENCH_GREY,
    height: iconSize,
    width: iconSize
  },
  focusedImage: {
    tintColor: CERULEAN
  }
```
And then in the `index.tsx`:
`<Image source={ICON} style={[styles.baseImage, focused && styles.focusedImage]} />`

Reply:
so I can apply an array of stylesheets instead of only one? and if theres a repeated attribute the last one will replace the first one?

Reply:
Exactly 😄